### ISSUE TITLE
Feature Request: Language selection for autoboot

### ISSUE THREAD

**Issue by henbagle** [ME3Tweaks/ME3TweaksModManager#269]
It would be really great to be able to select subtitle language and VO language when using the autoboot game feature. Opening up the launcher to choose these settings takes a lot of time, especially when I am testing every language to make sure something works.

**Comment by Mgamerz** [comment]
If you want to code something up that can make this change to that ini, I'd find way to include it (maybe via shift+start game)

**Comment by Mgamerz** [comment]
I've updated the bink2w64.dll code to support more arguments, but my C++ code is bad. Not sure how reliable it is, we have to have a rock solid dll in order to ship this to users.

**Comment by Mgamerz** [comment]
bink2w64.dll 2007 was shipped with 7.0.7 / 8.0 Beta 3. So far it hasn't blown anything up so we might be able to start working on passing options through that I built in. Unfortunately none of this works with EA app because they don't support command line arguments.

**Comment by mirh** [comment]
Sigh, it's such an horrible client, with bugs on every corner. 
Aside of that, after a truly atrocious amount of time poking the program, monitoring files and editing everything I could find, I finally managed to have it at least *forget* a game* is installed if you:
* stop EABackgroundService
* tamper with the .dat file in `%PROGRAMDATA%\Origin\LocalContent\*game-name*\`
* edit the `InstallDir` string inside `HKLM\SOFTWARE\WOW6432Node\Electronic Arts\*game-name*`
* delete `%PROGRAMDATA%\EA Desktop\*long-guid*\IS` 

Yes, that ain't actually saying anything about providing arguments, but if it's even possible the secret is probably inside the last mentioned cryptic file (the first three steps are fairly transparent and straightforward instead, and I don't think you'd be able to hide much like for example the executable name). 

But I found they store the friends list in `%LOCALAPPDATA%\Origin\Web Cache\data8` then, so far as much as I know they may be so dumb that they are downloading the equivalent of `installerdata.xml` on the fly from the web (which I guess would neatly link with the fact that indeed EA app doesn't work offline).

*I tried with ultima underworld, but I don't think this makes any difference to our purposes
